src/test/resources/firebase_config_bad_key.json

```diff
@@ -0,0 +1,4 @@
+{
+  "databaseUr1": "https://hipster-chat.firebaseio.mock",
```

Comment:
Use a different key (e.g. notValid) so the intention is clear. This still looks like databaseURL